constants_test.go

```diff
@@ -1,4 +1,4 @@
-package ofxgo_test
```

Comment:
I believe generate_constants.py should also be updated, since I'm lazy and use it to generate constants.go and constants_test.go if the constants change.